### ISSUE TITLE
Create streams codelab

### ISSUE THREAD

**Issue by kwalrath** [dart-lang/site-www#2030]
This would be a logical follow-on to the async-await/futures codelab.

**Comment by domesticmouse** [comment]
Hey @MaryaBelanger, can I re-assign this to you? This feels like it is intended to be a DartPad hosted codelab in the dart.dev codelabs section, not a full blown Flutter codelab on codelabs.developer.google.com.

**Comment by MaryaBelanger** [comment]
Yes definitely, content like this that's just meant for the site shouldn't be actual codelabs going forward. Thanks

**Comment by parlough** [comment]
I'm going to close this for now, as we likely won't create a codelab for this. Hopefully our in-progress getting started flow will cover it though.
